spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+describe UsersController do
+  context "can route to controller actions" do
+    it { is_expected.to route(:get, "/users")
+      .to(controller: :users, action: :index) }
+    it { is_expected.to route(:get, "/users/1")
+      .to(controller: :users, action: :show, id: 1) }
+    it { is_expected.to route(:get, "/users/new")
+      .to(controller: :users, action: :new) }
+    it { is_expected.to route(:post, "/users")
+      .to(controller: :users, action: :create) }
+    it { is_expected.to route(:get, "/users/1/edit")
+      .to(controller: :users, action: :edit, id: 1) }
+    it { is_expected.to route(:put, "/users/1")
+      .to(controller: :users, action: :update, id: 1) }
+    it { is_expected.to route(:delete, "/users/1")
+      .to(controller: :users, action: :destroy, id: 1) }
```

Comment:
A few thoughts about these:
1. They fit more in request specs
2. I wouldn't generally bother with adding them because their functionally is usually tested in other specs
3. Rubocop is correct. Multiline blocks should use `do..end` rather then `{ }`

Reply:
Thanks Elle, I will use the specs we did in class.

Reply:
And remove the exclude for the mulitine blocks and blockendnewline